src/main/resources/application.conf

```diff
@@ -89,6 +89,7 @@ workspace {
   entitiesPath="/workspaces/%s/%s/entities"
   entityQueryPath="/workspaces/%s/%s/entityQuery"
   importEntitiesPath="/workspaces/%s/%s/importEntities"
+  workspacesEditablePath="/workspaces/%s/%s"   //Maybe change this path name
```

Comment:
Nit: Paths can often be re-used for different operations, put, get, delete, etc. Make sure the variable name makes the most sense in context.